src/resources/information/en.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
```

Comment:
Just like with EPUB, I would recommend renaming the file extension from `.html` to `.xhtml`, to guarantee that the web browser engine triggers the polyglot XHTML5 parser/renderer.

Reply:
https://www.w3.org/publishing/epub32/epub-contentdocs.html#sec-xhtml-conf-content

> The XHTML Content Document filename SHOULD use the file extension .xhtml

Reply:
fixed bc925a57